near-sdk/src/environment/env.rs

```diff
@@ -200,8 +200,12 @@ pub fn current_contract_code() -> AccountContract {
 ///
 /// # Examples
 /// ```
+/// # #[cfg(feature = "deterministic-account-ids")]
```

Comment:
would it work if we just move `#[cfg(feature = "deterministic-account-ids")]` above the comment?